x-pack/functionbeat/docs/configuring-howto.asciidoc

```diff
@@ -0,0 +1,72 @@
+[id="configuring-howto-{beatname_lc}"]
+= Configuring {beatname_uc}
+
+[partintro]
+--
+Before modifying configuration settings, make sure you've completed the
+<<{beatname_lc}-configuration,configuration steps>> in the Getting Started.
+This section describes some common use cases for changing configuration options.
+
+include::{libbeat-dir}/docs/shared-configuring.asciidoc[]
+
+The following topics describe how to configure {beatname_uc}:
+
+* <<configuration-{beatname_lc}-options>>
+* <<configuration-general-options>>
+* <<configuring-internal-queue>>
+* <<configuring-output>>
+* <<configuration-ssl>>
+* <<filtering-and-enhancing-data>>
+* <<configuring-ingest-node>>
+* <<configuration-path>>
+* <<setup-kibana-endpoint>>
+* <<configuration-dashboards>>
+* <<configuration-template>>
+* <<configuration-logging>>
+* <<using-environ-vars>>
+* <<yaml-tips>>
+* <<regexp-support>>
+* <<http-endpoint>>
+* <<{beatname_lc}-reference-yml>>
+
+--
+
+include::./config-options.asciidoc[]
+
+include::./general-options.asciidoc[]
+
+:allplatforms:
+include::{libbeat-dir}/docs/queueconfig.asciidoc[]
+:allplatforms!:
+
+include::{libbeat-dir}/docs/outputconfig.asciidoc[]
+
+include::{libbeat-dir}/docs/shared-ssl-config.asciidoc[]
+
+include::./filtering.asciidoc[]
+
+include::{libbeat-dir}/docs/shared-config-ingest.asciidoc[]
+
+include::{libbeat-dir}/docs/shared-path-config.asciidoc[]
+
+include::{libbeat-dir}/docs/shared-kibana-config.asciidoc[]
+
+include::{libbeat-dir}/docs/setup-config.asciidoc[]
+
+include::{libbeat-dir}/docs/loggingconfig.asciidoc[]
```

Comment:
Maybe we should add a note that the logging will be in the cloudwatch log group for the lambda in your AWS panel.

Reply:
added to list ^^ of stuff to fix after merging.